docs/features/software-templates/writing-templates.md

```diff
@@ -523,6 +521,10 @@ output:
     - title: Open in catalog
       icon: catalog
       entityRef: ${{ steps['register'].output.entityRef }} # link to the entity that has been ingested to the catalog
+  text:
+    - title: More information
+      content: |
+        Access the \[remote repository\](${{ steps['publish'].output.remoteUrl }}).
```

Comment:
Hmm I think that this example isn't compatible with our tooling which checks that all links are valid. Might be a little bug there. I wonder if we can just do simpler example here for now so we can get this in the release and we can take a look at that bug which is making this fail at a later date.